src/client/DamlLedgerClient.ts

```diff
@@ -66,6 +66,33 @@ export class DamlLedgerClient implements LedgerClient {
     private readonly _partyManagementClient: PartyManagementClient;
     private readonly _packageManagementClient: PackageManagementClient;
 
+    private static readonly LedgerIdentityServiceClient =
```

Comment:
This is the most awkard part on the client side. Unfortunately, this seems to be the only way of handling this. Compare the official example at https://github.com/agreatfool/grpc_tools_node_protoc_ts/blob/af23508a32b19e6a65f41a01cd1096bb4810a783/examples/src/grpcjs/client.ts#L11.

Reply:
Note that this does not leak into the user-facing API.